.drone.yml

```diff
@@ -0,0 +1,1100 @@
+---
+kind: pipeline
+type: kubernetes
+name: test
+
+trigger:
+  branch:
+    - master
+    - branch/*
+  event:
+    exclude:
+      - promote
+      - rollback
+
+workspace:
+  path: /go
+
+clone:
+  disable: true
+
+steps:
+  - name: Check out code
+    image: golang:1.13.2
```

Comment:
Can this come from a variable? Otherwise we'll have to bump the version of Go in many different places.

Reply:
Sort of. The problem is that the `golang` container tags are in the `1.13.2` format but all our own `RUNTIME` references (which we use to tag our `buildbox` containers and pass into builds etc) are in `go1.13.2` format. I'll see whether I can do some string manipulation to get it right.

The best I can manage is that we define `RUNTIME` once at the top of each pipeline, which will be 7 or 8 copies throughout the whole file.

Reply:
This is a no-can-do because Drone doesn't seem to allow you to use variables in image names for pipeline steps.

To be honest, changing the version of Go should be as simple as a global search-and-replace on this file to change `1.13.2` to `1.14.4`. If we were going to bump the version of Go then we'd need to change it throughout the repo with a global search and replace anyway so I don't see it causing any trouble.